src/sentry/migrations/0926_dashboard_favorite_defer_position_constraint.py

```diff
@@ -0,0 +1,41 @@
+# Generated by Django 5.2.1 on 2025-06-10 14:57
+
+import django.db.models.constraints
+from django.db import migrations, models
+
+from sentry.new_migrations.migrations import CheckedMigration
+
+
+class Migration(CheckedMigration):
+    # This flag is used to mark that a migration shouldn't be automatically run in production.
+    # This should only be used for operations where it's safe to run the migration after your
+    # code has deployed. So this should not be used for most operations that alter the schema
+    # of a table.
+    # Here are some things that make sense to mark as post deployment:
+    # - Large data migrations. Typically we want these to be run manually so that they can be
+    #   monitored and not block the deploy for a long period of time while they run.
+    # - Adding indexes to large tables. Since this can take a long time, we'd generally prefer to
+    #   run this outside deployments so that we don't block them. Note that while adding an index
+    #   is a schema change, it's completely safe to run the operation after the code has deployed.
+    # Once deployed, run these manually via: https://develop.sentry.dev/database-migrations/#migration-deployment
+
+    is_post_deployment = False
+
+    dependencies = [
+        ("sentry", "0925_backfill_open_periods"),
+    ]
+
+    operations = [
+        migrations.RemoveConstraint(
+            model_name="dashboardfavoriteuser",
+            name="sentry_dashboardfavoriteuser_user_id_organization_id_position_uniq",
+        ),
+        migrations.AddConstraint(
+            model_name="dashboardfavoriteuser",
+            constraint=models.UniqueConstraint(
+                deferrable=django.db.models.constraints.Deferrable["DEFERRED"],
+                fields=("user_id", "organization_id", "position"),
+                name="sentry_dashboardfavoriteuser_user_id_organization_id_position_uniq_deferred",
```

Comment:
@markstory I merged this this morning and noticed the deploy pipeline failed complaining about a duplicate table r.e. the constraint index

https://deploy.getsentry.net/go/tab/build/detail/deploy-getsentry-backend-s4s/12615/migrations/1/migrations

I'm not entirely sure why the pipeline didn't catch it earlier but since I've changed the name, this should resolve the conflict right?